validator/client/service.go

```diff
@@ -379,3 +387,30 @@ func (v *ValidatorService) DeleteGraffiti(ctx context.Context, pubKey [fieldpara
 	}
 	return v.validator.DeleteGraffiti(ctx, pubKey)
 }
+
+func runNodeSwitcherRoutine(ctx context.Context, v iface.Validator, endpoints []string) {
```

Comment:
I think this needs to be removed and some of the logic from this should be added to 
https://github.com/prysmaticlabs/prysm/blob/d19365507f03ef5e83ae3b4aeeb42c459d5759f7/validator/client/runner.go#L296 instead